### PR TITLE
Feature -> Endpoint that retrieves all channels

### PR DESCRIPTION
**Pull request by ManasesLovera**
Endpoint that retrieves all channels was created based on the discord-proxy module, let me know if anything should be modified or fixed.

### Created: 
1. channel.services
2. channel.controller

### Modified:
1. channel.module

### Deleted:
1. dto/
2. schemas/

> Based on discord-proxy.services

### CODE REVIEW

**Review by stealkiller06 — CHANGES_REQUESTED** [review]
Remember what we talked.
Channel is part of the discord feature, that's why channel needs to be inside discord-proxy

**Comment by ManasesLovera** [comment]
Please review now

File: src/app.module.ts [hunk 1]
```diff
@@ -9,11 +9,13 @@ import { MongooseModule } from '@nestjs/mongoose';
 import { ExampleModule } from './example/example.module';
 import { ChallengeTrackerModule } from './features/challenge-tracker/challenge-tracker.module';
 import { DiscordProxyModule } from './features/discord-proxy/discord-proxy.module';
+import { ChannelModule } from './channel/channel.module';
 
 @Module({
   imports: [
     MongooseModule.forRoot('mongodb://localhost/discord_bot'),
     ExampleModule,
+    ChannelModule,
```
Comment (stealkiller06) [inline src/app.module.ts:18]: We don't need to import ChannelModule in app.module
Channel module is part of DiscordModule, so you just need to import it in DiscordModule and that's it 
See the example that we have in the ChallengeTrackerModule.

**Review by stealkiller06 — CHANGES_REQUESTED** [review]

**Comment by ManasesLovera** [comment]
Done!

**Review by stealkiller06 — APPROVED** [review]
LGTM

**Pull request merged**